helpers/useSession.tsx

```diff
@@ -1,23 +1,37 @@
 import useSWR from 'swr'
 import { User } from './models/User'
+import _ from 'lodash'
 
 const SERVER_URL = process.env.SERVER_URL
 
 export const fetcher = (url: string) =>
-  fetch(url, { credentials: 'include' }).then(r => r.json())
+  fetch(url, {
```

Comment:
Try using this instead: 

```
import { request } from 'graphql-request'

const API = 'https://api.graph.cool/simple/v1/movies'
const fetcher = query => request(API, query)

function App () {
  const { data, error } = useSWR(
    `{
      Movie(title: "Inception") {
        releaseDate
        actors {
          name
        }
      }
    }`,
    fetcher
  )
  // ...
}
```

source: https://github.com/zeit/swr#data-fetching

Reply:
We are going to refactor `context` out. So this can be resolved.